renovate.json

```diff
@@ -9,6 +9,12 @@
   "commitMessageExtra": "from {{{currentValue}}} to {{#if isMajor}}{{{newMajor}}}{{else}}{{#if isSingleVersion}}{{{newVersion}}}{{else}}{{{newValue}}}{{/if}}{{/if}}",
   "commitBody": "{{{releaseNotes}}}",
   "packageRules": [
+    {
+      "matchManagers": ["gradle"],
+      "extends": [":semanticCommitTypeAll(fix)"],
+      "groupName": null,
```

Comment:
I don't want to use grouping for real dependencies, because that would result in a single commit which does not mention the updated dependency. Imho that is not good for our changelog.